crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstnerBatched.cs

```diff
@@ -853,7 +855,12 @@ public bool Validate(OceanRenderer ocean, ValidatedHelper.ShowMessage showMessag
             // Renderer
             if (_mode == GerstnerMode.Geometry)
             {
-                isValid = ValidatedHelper.ValidateRenderer(gameObject, "Crest/Inputs/Animated Waves/Gerstner", showMessage);
+                isValid = ValidatedHelper.ValidateInputMesh(true, gameObject, showMessage);
+
+                if (TryGetComponent<MeshRenderer>(out var meshRenderer))
+                {
+                    isValid = ValidatedHelper.ValidateMaterial(meshRenderer.sharedMaterial, "Crest/Inputs/Animated Waves/Gerstner", gameObject, ValidatedHelper.DebugLog) && isValid;
+                }
```

Comment:
Shouldn't this remain as _ValidateRenderer_ as it doesn't support splines?

Reply:
good point done